api/swagger.yaml

```diff
@@ -1610,6 +1610,13 @@ definitions:
       ID:
         type: "string"
       Version:
+        description: |
+          The version number of the node object. This is required to avoid conflicting writes.
+          For every node object, the version number is incremented each time it is updated. It is required
```

Comment:
The actual behavior of this field is implementation-specific. From a client perspective, the only thing that matters is the the update _may not_ be applied if the version field has changed from the last read.